src/main/java/hyperledger/besu/java/rest/client/filters/BlockFilterHandler.java

```diff
@@ -0,0 +1,27 @@
+package hyperledger.besu.java.rest.client.filters;
+
+import hyperledger.besu.java.rest.client.config.EthConfig;
+import hyperledger.besu.java.rest.client.service.EventPublishService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BlockFilterHandler {
+  @Autowired private EthConfig ethConfig;
+
+  @Autowired(required = false)
+  private EventPublishService eventPublishServiceImpl;
+
+  // To receive all new blocks as they are added to the blockchain (the false parameter specifies
+  // that we only want the blocks, not the embedded transactions too):
+  public void receiveNewlyAddedBlocksOnly() {
+    ethConfig
+        .getWeb3jList()
+        .get(0)
+        .blockFlowable(false)
```

Comment:
why we changed it to false, earlier it was set as true ?

Reply:
corrected!